docs/products/flink/howto/rest-api-cheatsheet.rst

```diff
@@ -0,0 +1,82 @@
+Flink REST API examples cheatsheet
+==================================
+
+This page contains an overview of the REST API calls examples useful to create Aiven for Apache Flink table and job definitions.
+
+Create PostgreSQL based Flink table
+-------------------------------------------------
+
+Define a Flink table named ``node_info``
+
+* pointing to a PostgreSQL table named ``public.node_definition`` available in the PostgreSQL database identified by the integration with the ID ``a4af7409-d167-4f31-af13-ddd4cd06564f``. 
```

Comment:
For sample data, I think realistic but obviously false would work well. So using `aaaaaaaa-1111-2222-3333-bbbbbbbbbbbb` or something, consistently for each of the resources you're referring to, might make this easier to follow.